esp-hal/src/spi/mod.rs

```diff
@@ -40,7 +40,7 @@ pub enum Error {
 }
 
 #[doc(hidden)]
-#[cfg(any(doc, feature = "unstable"))]
+#[instability::unstable]
```

Comment:
I would argue this can be `#[cfg(feature = "unstable")]` We don't need to document a hidden impl, whether it's stable or not.

Reply:
True, thanks!